src/pages/RadicalCollaboration/PhotoCardMobile.js

```diff
@@ -41,7 +41,7 @@ const PhotoCardMobile = (props) => {
                 </CardContent>
               </CardActionArea>
               <CardActions className={classes.mobileCardActions}>
-                <Button className={classes.mobileButton}>
+                <Button href={i.route} className={classes.mobileButton}>
```

Comment:
Just a minor suggestion. For code readability, we might probably want to replace the meaningless variable 'i' with something else.

Reply:
Good suggestion. It was meant to represent an item from items.props, but got totally cryptic. I changed 'i' to 'item'.